app/res/layout/audio_prototype.xml

```diff
@@ -1,67 +1,86 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
 
-    android:orientation="vertical"
     android:layout_width="fill_parent"
     android:layout_height="fill_parent"
     android:layout_gravity="center_horizontal"
     android:gravity="center_horizontal"
-    android:paddingTop="10dp"
-    android:paddingLeft="10dp"
-    android:paddingRight="10dp"
+    android:orientation="vertical"
     android:paddingStart="10dp"
-    android:paddingEnd="10dp">
+    android:paddingLeft="10dp"
+    android:paddingTop="10dp"
+    android:paddingEnd="10dp"
+    android:paddingRight="10dp">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:layout_gravity="center"
-        android:gravity="center"
-        android:layout_marginTop="10dp">
+        android:layout_marginTop="10dp"
+        android:orientation="horizontal">
 
         <ImageButton
+            android:id="@+id/play_audio"
             android:layout_width="65dp"
             android:layout_height="65dp"
-            android:background="@drawable/play_disabled"
-            android:id="@+id/play_audio"
+            android:layout_marginEnd="10dp"
             android:layout_marginRight="10dp"
-            android:elevation="10dp"
-            android:layout_marginEnd="10dp" />
+            android:background="@drawable/play_disabled"
+            android:elevation="10dp" />
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/2aa03aecc11ade0381c1f426b782e030115c4729/app/res/layout/audio_prototype.xml#L29"></span>Attribute <code>elevation</code> is only used in API level 21 and higher (current min is 16)</td>
    </tr>
  </tbody>
</table>